clients/cli/src/main/java/org/apache/gravitino/cli/commands/ModelDetails.java

```diff
@@ -0,0 +1,92 @@
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.cli.commands;
+
+import java.util.Arrays;
+import org.apache.gravitino.NameIdentifier;
+import org.apache.gravitino.cli.ErrorMessages;
+import org.apache.gravitino.client.GravitinoClient;
+import org.apache.gravitino.exceptions.NoSuchCatalogException;
+import org.apache.gravitino.exceptions.NoSuchMetalakeException;
+import org.apache.gravitino.exceptions.NoSuchModelException;
+import org.apache.gravitino.exceptions.NoSuchSchemaException;
+import org.apache.gravitino.model.Model;
+import org.apache.gravitino.model.ModelCatalog;
+
+/** Displays the details of a model. */
+public class ModelDetails extends Command {
+  protected final String metalake;
+  protected final String catalog;
+  protected final String schema;
+  protected final String model;
+
+  /**
+   * Displays the details of a model.
+   *
+   * @param url The URL of the Gravitino server.
+   * @param ignoreVersions If true don't check the client/server versions match.
+   * @param metalake The name of the metalake.
+   * @param catalog The name of the catalog.
+   * @param schema The name of schema.
+   * @param model The name of model.
+   */
+  public ModelDetails(
+      String url,
+      boolean ignoreVersions,
+      String metalake,
+      String catalog,
+      String schema,
+      String model) {
+    super(url, ignoreVersions);
+    this.metalake = metalake;
+    this.catalog = catalog;
+    this.schema = schema;
+    this.model = model;
+  }
+
+  /** Displays the details of a model. */
+  @Override
+  public void handle() {
+    NameIdentifier name = NameIdentifier.of(schema, model);
+    Model gModel = null;
+    int[] versions = new int[0];
+
+    try {
+      GravitinoClient client = buildClient(metalake);
+      ModelCatalog modelCatalog = client.loadCatalog(catalog).asModelCatalog();
+      gModel = modelCatalog.getModel(name);
+      versions = modelCatalog.listModelVersions(name);
+    } catch (NoSuchMetalakeException noSuchMetalakeException) {
+      exitWithError(ErrorMessages.UNKNOWN_METALAKE);
+    } catch (NoSuchCatalogException noSuchCatalogException) {
+      exitWithError(ErrorMessages.UNKNOWN_CATALOG);
+    } catch (NoSuchSchemaException noSuchSchemaException) {
+      exitWithError(ErrorMessages.UNKNOWN_SCHEMA);
+    } catch (NoSuchModelException noSuchModelException) {
+      exitWithError(ErrorMessages.UNKNOWN_MODEL);
+    } catch (Exception err) {
+      exitWithError(err.getMessage());
+    }
+    String basicInfo =
+        String.format("Model name %s, latest version: %s%n", gModel.name(), gModel.latestVersion());
+    String versionInfo = Arrays.toString(versions);
+    System.out.printf(basicInfo + "versions: " + versionInfo);
```

Comment:
This is a little inconsistent with the output of other commands and could done with some minor tweaks.